### PR TITLE
[LottieGen] Added new AnimationController API support for LottieGen

### PR DESCRIPTION
**Pull request by aborziak-ms**
Instead of binding Progress to each individual `AnimationController` in this change we are creating one `AnimationController` for entire animation and then using it for all animations.

Note: Lottie representation of `AnimationController` needs to have boolen flag `"IsCustom"` which displays how the `AnimationController` was created - as a custom one (using `Compositor::CreateAnimationController`), or was obtained from an object with `TryGetAnimationController`. Only custom controller can be used in `StartAnimation(property, animation, controller)` calls to make it easier to track the lifetime of `AnimationController` objects and to make it easier to generate the code.

### CODE REVIEW

File: source/LottieToWinComp/CompositionObjectFactory.cs [hunk 1]
```diff
@@ -86,6 +86,9 @@ internal uint GetUapVersionForApi(string apiName)
                 case nameof(PathKeyFrameAnimation):
                     return 11;
 
+                case nameof(AnimationController):
```
Comment (raymond-wh-leung) [inline source/LottieToWinComp/CompositionObjectFactory.cs:89]: >case nameof(AnimationController): [](http://example.com/codeflow?start=16&length=33)

nit: how about add a comment that, this class was introduced in version 6 (to control one keyframeAnimator); the support for it to control multiple keyframeAnimators was added in version 14.

**Review by raymond-wh-leung — APPROVED** [review]
:shipit:

**Review by raymond-wh-leung — APPROVED** [review]
:shipit:

**Review by raymond-wh-leung — DISMISSED** [review]
:shipit:

**Comment by raymond-wh-leung** [comment]
btw, saw the commit description "API UAP 14", for SV2 the UAP version should be 15.

**Comment by aborziak-ms** [comment]
@raymond-wh-leung oh right, I will edit it. 
Trying to fix the build now

**Comment by raymond-wh-leung** [comment]
I see gotcha!

---
In reply to: [1139124839](https://github.com/CommunityToolkit/Lottie-Windows/pull/488#issuecomment-1139124839) [](http://example.com/codeflow?ancestors=1139124839,1139124839)

**Comment by raymond-wh-leung** [comment]
>         <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>

Is it by design that it is reverted from version 10.0.22558.0 from iteration #3

---
Refers to: Directory.Build.props:63 in d8a80eb. [](commit_id = d8a80eb4950bdcdeab6eb29606be078b93d840ed, deletion_comment = False)

**Pull request merged**